components/Contact/molecules/ImageSlider.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+import Slider, { Settings as SlickSettings } from 'react-slick';
+import Image from 'next/image';
+
+import Loc1Image from '@/public/assets/location-1.png';
+import Loc2Image from '@/public/assets/location-2.png';
+import Loc3Image from '@/public/assets/location-3.png';
+import Loc4Image from '@/public/assets/location-4.png';
+
+const PsychoSlider = styled(Slider)`
+  overflow-x: hidden;
+  margin: 0 -40px;
+
+  .slick-track {
+    display: flex;
+    justify-content: center;
+  }
+  
+  @media screen and (min-width: 664px) {
+    margin: 0;
+
+    .slick-track {
+      justify-content: start;
+    }
```

Comment:
there is a missing bracket here, try the ```vscode-styled-components``` plugin, it helps a lot during development